packages/devtools_app/lib/src/table_data.dart

```diff
@@ -393,6 +395,9 @@ abstract class ColumnData<T> {
   /// Get the cell's tooltip value from the given [dataObject].
   String getTooltip(T dataObject) => getDisplayValue(dataObject);
 
+  /// Get the cell's text color from the given [dataObject].
+  Color getTextColor(T dataObject) => null;
```

Comment:
not your fault but add a todo to the top of this class that says "remove get from method names"